js/ui/text_box/ui.text_editor.mask.strategy.base.js

```diff
@@ -129,16 +129,38 @@ export default class BaseMaskStrategy {
         const caret = this.editorCaret();
 
         editor._maskKeyHandler(event, () => {
-            const pastingText = getClipboardText(event);
+            const pastedText = getClipboardText(event);
             const restText = editor._maskRulesChain.text().substring(caret.end);
-            const accepted = editor._handleChain({ text: pastingText, start: caret.start, length: pastingText.length });
+            const accepted = editor._handleChain({ text: pastedText, start: caret.start, length: pastedText.length });
             const newCaret = caret.start + accepted;
 
             editor._handleChain({ text: restText, start: newCaret, length: restText.length });
             editor._caret({ start: newCaret, end: newCaret });
         });
     }
 
+    _autoFillHandler(event) {
+        const { editor } = this;
+        const inputVal = this.editorInput().val();
+        this._inputHandlerTimer = setTimeout(() => {
+            this._keyPressHandled = true;
+
+            if(this._isAutoFill(event)) {
+                this._keyPressHandled = true;
+
+                editor._maskKeyHandler(event, () => {
+                    editor._handleChain({ text: inputVal, start: 0, length: inputVal.length });
+                });
+                editor._validateMask();
+            }
+        });
+    }
+
+    _isAutoFill() {
+        const input = this.editor._input().get(0);
+        return input && input.matches(':-webkit-autofill');
```

Comment:
Fails in IE:
`TypeError: Object doesn't support property or method 'matches'`
https://github.com/alexander-kotov-dx/DevExtreme/runs/528109765?check_suite_focus=true

and FF:
`':-webkit-autofill' is not a valid selector`
https://devextreme-ci.devexpress.com/DevExpress/DevExtreme/29719/108